runtime/common/NoiseModel.h

```diff
@@ -63,12 +64,25 @@ struct kraus_op {
   }
 
   /// @brief Set this kraus_op equal to the other
-  kraus_op &operator=(const kraus_op &other);
+  kraus_op &operator=(const kraus_op &other) {
+    data = other.data;
+    return *this;
+  }
 
   /// @brief Return the adjoint of this kraus_op
-  kraus_op adjoint();
+  kraus_op adjoint() const {
+    std::size_t N = data.size();
+    std::vector<cudaq::complex> newData(N);
+    for (std::size_t i = 0; i < N; i++)
+      for (std::size_t j = 0; j < N; j++)
+        newData[j * N + i] = std::conj(data[i * N + j]);
```

Comment:
Not sure if it's a bug but it seems like we're indexing beyond the length (`N`) of `newData`.
(perhaps the loop bounds are `nRows` and `nCols`?)

Reply:
My bad. Will fix.